modules/labs-react/text-input/stories/examples/ThemedAlert.tsx

```diff
@@ -13,7 +13,7 @@ export const ThemedAlert = () => {
           focusOutline: colors.grapeSoda300,
         },
         alert: {
-          main: colors.kiwi500,
+          main: colors.kiwi200,
```

Comment:
Updated this example to show what happens if your main color doesn't pass contrast.

Reply:
Cool. Let's make a note about that in the mdx doc, so it's clear what's happening.